posthog/async_migrations/migrations/0005_person_replacing_by_version.py

```diff
@@ -70,6 +70,9 @@ class Migration(AsyncMigrationDefinition):
 
     depends_on = "0004_replicated_schema"
 
+    posthog_min_version = "1.38.0"
```

Comment:
What function does min_version serve? We introduced this in 1.38.0 for end users, but making this mandatory just adds complications for running anything on cloud.

Reply:
discussed in sync